x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -0,0 +1,204 @@
+      {
+        setServerData: (_: IOverviewValues, { fpAccount }: IOverviewValues) => fpAccount,
+      },
+    ],
+    canCreateInvitations: [
+      false,
+      {
+        setServerData: (_: IOverviewValues, { canCreateInvitations }: IOverviewValues) =>
+          canCreateInvitations,
+      },
+    ],
+    flashMessages: [
+      {},
+      {
+        setFlashMessages: (_: IOverviewValues, { flashMessages }: IOverviewValues) => flashMessages,
+      },
+    ],
+    hasUsers: [
+      false,
+      {
+        setServerData: (_: IOverviewServerData, { hasUsers }: IOverviewValues) => hasUsers,
+      },
+    ],
+    hasOrgSources: [
+      false,
+      {
+        setServerData: (_: IOverviewServerData, { hasOrgSources }: IOverviewValues) =>
+          hasOrgSources,
+      },
+    ],
+    canCreateContentSources: [
+      false,
+      {
+        setServerData: (_: IOverviewServerData, { canCreateContentSources }: IOverviewValues) =>
+          canCreateContentSources,
+      },
+    ],
+    isOldAccount: [
+      false,
+      {
+        setServerData: (_: IOverviewServerData, { isOldAccount }: IOverviewValues) => isOldAccount,
+      },
+    ],
+    sourcesCount: [
+      0,
+      {
+        setServerData: (_: IOverviewServerData, { sourcesCount }: IOverviewValues) => sourcesCount,
+      },
+    ],
+    pendingInvitationsCount: [
+      0,
+      {
+        setServerData: (_: IOverviewServerData, { pendingInvitationsCount }: IOverviewValues) =>
+          pendingInvitationsCount,
+      },
+    ],
+    accountsCount: [
+      0,
+      {
+        setServerData: (_: IOverviewServerData, { accountsCount }: IOverviewValues) =>
+          accountsCount,
+      },
+    ],
+    personalSourcesCount: [
+      0,
+      {
+        setServerData: (_: IOverviewServerData, { personalSourcesCount }: IOverviewValues) =>
+          personalSourcesCount,
+      },
+    ],
+    activityFeed: [
+      [],
+      {
+        setServerData: (_: IOverviewServerData, { activityFeed }: IOverviewValues) => activityFeed,
+      },
+    ],
+    dataLoading: [
+      true,
+      {
+        setServerData: () => false,
+        setHasErrorConnecting: () => false,
+      },
+    ],
+    hasErrorConnecting: [
+      false,
+      {
+        setHasErrorConnecting: (_: IOverviewServerData, { hasErrorConnecting }: IOverviewValues) =>
+          hasErrorConnecting,
+      },
+    ],
+  }),
+  selectors: ({ selectors }: { selectors: IOverviewValues }) => ({
+    hideOnboarding: [
+      () => [
+        selectors.hasUsers,
+        selectors.hasOrgSources,
+        selectors.isOldAccount,
+        selectors.organization,
+      ],
+      (hasUsers: boolean, hasOrgSources: boolean, isOldAccount: boolean, org: IOrganization) =>
+        hasUsers && hasOrgSources && org.name !== org.defaultOrgName && isOldAccount,
+    ],
+    statsColumns: [
+      () => [selectors.isFederatedAuth],
+      (isFederatedAuth: boolean) => (isFederatedAuth ? 'halves' : 'fourths'),
+    ],
```

Comment:
Are we actually using this var anywhere? I don't see it being used by a view, I thought we simplified it to a much easier to read `<EuiFlexGrid columns={isFederatedAuth ? 1 : 2}>`

```suggestion
```